lib/generators/step/templates/show/show.html.erb

```diff
@@ -1,17 +1,15 @@
 <%% title t('.page_title') %>
+<%%= step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%%=t '.heading' %></h1>
+    <h1 class="govuk-heading-xl"><%%=t '.heading' %></h1>
 
-    <p class="lede"><%%=t '.lead_text' %></p>
+    <p class="govuk-body-l"><%%=t '.lead_text' %></p>
 
     <!-- TODO: Add the copy needed, and adapt the button linking to another step -->
 
-    <div class="xform-group form-submit">
-      <%%= link_to t('.continue'), '#', class: 'button', role: 'button' %>
-    </div>
+    <%%= link_to t('.continue'), '#', class: 'govuk-button' %>
```

Comment:
This can make use of the helper, `link_button t('.continue'), '#'`
It will generate the proper classes and markup.